test-requirements.txt

```diff
@@ -2,3 +2,5 @@ coverage
 glueviz
 matplotlib
 pytest
+vcrpy
```

Comment:
Note that this list is alphabetized to make it easy to check whether something is present. (It's admittedly of little use in such as short list, but it's our system for all the projects.)